<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-17T10:57:56 NPT" name="Default suite" finished-at="2022-09-17T10:58:24 NPT" duration-ms="28490">
    <groups>
    </groups>
    <test started-at="2022-09-17T10:57:56 NPT" name="Default test" finished-at="2022-09-17T10:58:24 NPT" duration-ms="28490">
      <class name="tests.LoginPageTest">
        <test-method is-config="true" signature="extendReportSetup()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:57:56 NPT" name="extendReportSetup" finished-at="2022-09-17T10:57:56 NPT" duration-ms="165" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extendReportSetup -->
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:57:56 NPT" name="setup" finished-at="2022-09-17T10:58:07 NPT" duration-ms="10983" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginPageLoginFunctionalityTestCase()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:58:07 NPT" name="loginPageLoginFunctionalityTestCase" finished-at="2022-09-17T10:58:12 NPT" duration-ms="5177" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageLoginFunctionalityTestCase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:58:12 NPT" name="tearDown" finished-at="2022-09-17T10:58:12 NPT" duration-ms="425" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:58:12 NPT" name="setup" finished-at="2022-09-17T10:58:23 NPT" duration-ms="10747" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginPageLogoTestCase()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:58:23 NPT" name="loginPageLogoTestCase" finished-at="2022-09-17T10:58:24 NPT" duration-ms="785" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertFalse(Assert.java:78)
at org.testng.Assert.assertFalse(Assert.java:88)
at tests.LoginPageTest.loginPageLogoTestCase(LoginPageTest.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageLogoTestCase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:58:24 NPT" name="tearDown" finished-at="2022-09-17T10:58:24 NPT" duration-ms="351" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="flushExtentReport()[pri:0, instance:tests.LoginPageTest@771a660]" started-at="2022-09-17T10:58:24 NPT" name="flushExtentReport" finished-at="2022-09-17T10:58:24 NPT" duration-ms="217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushExtentReport -->
      </class> <!-- tests.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
